templates/common/paths.go

```diff
@@ -22,7 +22,10 @@ import (
 // ABCInternalDir is the name for internal directories that have things like
 // manifests and .gitkeep files. These may be destination directories or
 // golden-test data/ directories.
-const ABCInternalDir = ".abc"
+const (
+	ABCInternalDir    = ".abc"
+	ABCInternalStdout = ".stdout"
```

Comment:
I don't think this name should start with a dot. Typically that's for "hidden" files, but this file is already in a hidden directory so it doesn't need to be hidden. For example, if you look in `~/.vscode` and `~/.config`, most files in there don't start with a dot.

Reply:
fixed by https://github.com/abcxyz/abc/pull/419